exchanges/coinut/coinut_wrapper.go

```diff
@@ -721,22 +721,22 @@ func (c *COINUT) GetActiveOrders(req *order.GetOrdersRequest) ([]order.Detail, e
 				orders = append(orders, order.Detail{
 					Exchange:        c.Name,
 					ID:              strconv.FormatInt(openOrders.Orders[i].OrderID, 10),
-					CurrencyPair:    c.FormatExchangeCurrency(currency.NewPairFromString(currenciesToCheck[x]), asset.Spot),
-					OrderSide:       order.Side(openOrders.Orders[i].Side),
-					OrderDate:       time.Unix(0, openOrders.Orders[i].Timestamp),
+					Pair:            c.FormatExchangeCurrency(currency.NewPairFromString(currenciesToCheck[x]), asset.Spot),
+					Side:            order.Side(openOrders.Orders[i].Side),
+					Date:            time.Unix(0, openOrders.Orders[i].Timestamp),
 					Status:          order.Active,
 					Price:           openOrders.Orders[i].Price,
-					Amount:          openOrders.Orders[i].Qty,
-					ExecutedAmount:  openOrders.Orders[i].Qty - openOrders.Orders[i].OpenQty,
-					RemainingAmount: openOrders.Orders[i].OpenQty,
+					Amount:          openOrders.Orders[i].Quantity,
+					ExecutedAmount:  openOrders.Orders[i].Quantity - openOrders.Orders[i].OpenQuantity,
+					RemainingAmount: openOrders.Orders[i].OpenQuantity,
 				})
 			}
 		}
 	} else {
 		var instrumentsToUse []int64
-		if len(req.Currencies) > 0 {
-			for x := range req.Currencies {
-				curr := c.FormatExchangeCurrency(req.Currencies[x],
+		if len(req.Pairs) > 0 {
```

Comment:
Can drop this if check, since a range will effectively do the same thing and instantly continue if it's 0